src/components/Quiz/QuizzesStats.tsx

```diff
@@ -41,25 +49,82 @@ const handleShare = ({ score, total }: QuizShareStats) => {
   })
 }
 
+const getFormattedStats = (
+  language,
+  average,
+  collectiveQuestionsAnswered,
+  collectiveAverageScore,
+  collectiveRetryRate
+) => {
+  const localeForNumbers = getLocaleForNumberFormat(language as Lang)
+
+  // Initialize number and percent formatters
+  const numberFormatter = new Intl.NumberFormat(localeForNumbers, {
+    style: "decimal",
+    minimumSignificantDigits: 2,
+    maximumSignificantDigits: 3,
+  })
+
+  const percentFormatter = new Intl.NumberFormat(localeForNumbers, {
+    style: "percent",
+    minimumSignificantDigits: 2,
+    maximumSignificantDigits: 3,
```

Comment:
- Instead of showing single digit percentages (0-9%) as x.0% (2 significant digits) show as 0%
- Instead of showing single digit numbers (0-9) as x.0 (2 significant digits) show an integer 

```suggestion
  const numberFormatter = new Intl.NumberFormat(localeForNumbers, {
    style: "decimal",
    minimumSignificantDigits: 1,
    maximumSignificantDigits: 3,
  })

  const percentFormatter = new Intl.NumberFormat(localeForNumbers, {
    style: "percent",
    minimumSignificantDigits: 1,
    maximumSignificantDigits: 3,
```